Packages/io.chainsafe.web3-unity/Samples~/Web3.Unity/Scripts/Scenes/SampleMain/Unsorted/ContractSendBehaviour.cs

```diff
@@ -8,6 +8,10 @@ public class ContractSendBehaviour : SampleBehaviour
         public string method = "addTotal";
         public string abi = "[ { \"inputs\": [ { \"internalType\": \"uint8\", \"name\": \"_myArg\", \"type\": \"uint8\" } ], \"name\": \"addTotal\", \"outputs\": [], \"stateMutability\": \"nonpayable\", \"type\": \"function\" }, { \"inputs\": [], \"name\": \"myTotal\", \"outputs\": [ { \"internalType\": \"uint256\", \"name\": \"\", \"type\": \"uint256\" } ], \"stateMutability\": \"view\", \"type\": \"function\" } ]";
         public string contractAddress = "0x7286Cf0F6E80014ea75Dbc25F545A3be90F4904F";
+        public object[] args =
+        {
+            1
```

Comment:
What does that number 1 actually represent?

Reply:
The amount to increase the mapping by like so:

`// SPDX-License-Identifier: MIT
pragma solidity ^0.8.19;

contract AddTotal {
    uint256 public myTotal = 0;

    function addTotal(uint8 _myArg) public {
        myTotal = myTotal + _myArg;
    }
}`

Reply:
I should probably refactor the object so it initialized from a public integer so it's not sitting there as a magic number too. BRB.

Reply:
done